examples/rounded-cylinder.fixture.tsx

```diff
@@ -3,7 +3,7 @@ import { RoundedCylinder } from "../lib"
 import { JsCadView } from "../lib/components/jscad-view"
 
 export default () => (
-  <ExampleWrapper>
+  <ExampleWrapper fileName="rounded-cylinder.fixture.tsx">
```

Comment:
can you try using `import.meta.path`

Reply:
<img width="594" alt="Screenshot 2024-11-23 at 1 09 58 AM" src="https://github.com/user-attachments/assets/791ec432-fa81-479e-a901-3d7798028839">

Reply:
<img width="712" alt="Screenshot 2024-11-23 at 1 19 36 AM" src="https://github.com/user-attachments/assets/73f6c02b-28d5-4fe3-80a3-88a0d8f8789e">